core/src/main/java/hudson/model/Computer.java

```diff
@@ -321,7 +321,7 @@ public WorkspaceList getWorkspaceList() {
      * Gets the string representation of the agent log.
      */
     public String getLog() throws IOException {
-        return Util.loadFile(getLogFile());
+        return Util.loadFile(getLogFile(), Charset.defaultCharset());
```

Comment:
Matches `getLogText()` so OK for now, though ultimately I think this is wrong: logs should be in UTF-8, which https://github.com/jenkinsci/jenkins/blob/7edfc7170c56780ccc715ed67cbe58a37d65bafb/core/src/main/java/hudson/slaves/SlaveComputer.java#L156 ought to enforce.

Reply:
```suggestion
        return Util.loadFile(getLogFile(), /* TODO switch agent logs to UTF-8 */ Charset.defaultCharset());
```